### ISSUE TITLE
Broken URL

### ISSUE THREAD

**Issue by rogershijin** [zaemyung/crawl-reuters#6]
Hi I tried running this and I got the following printout:

2018-12-07 12:15:33 [scrapy.utils.log] INFO: Scrapy 1.5.1 started (bot: crawler)
2018-12-07 12:15:33 [scrapy.utils.log] INFO: Versions: lxml 4.2.5.0, libxml2 2.9.8, cssselect 1.0.3, parsel 1.5.1, w3lib 1.19.0, Twisted 18.7.0, Python 3.7.0 (default, Jun 28 2018, 13:15:42) - [GCC 7.2.0], pyOpenSSL 18.0.0 (OpenSSL 1.0.2p  14 Aug 2018), cryptography 2.3.1, Platform Linux-4.16.13-2-ARCH-x86_64-with-arch
2018-12-07 12:15:33 [scrapy.crawler] INFO: Overridden settings: {'BOT_NAME': 'crawler', 'NEWSPIDER_MODULE': 'crawler.spiders', 'ROBOTSTXT_OBEY': True, 'SPIDER_MODULES': ['crawler.spiders']}
2018-12-07 12:15:34 [scrapy.middleware] INFO: Enabled extensions:
['scrapy.extensions.corestats.CoreStats',
 'scrapy.extensions.telnet.TelnetConsole',
 'scrapy.extensions.memusage.MemoryUsage',
 'scrapy.extensions.logstats.LogStats']
2018-12-07 12:15:34 [scrapy.middleware] INFO: Enabled downloader middlewares:
['scrapy.downloadermiddlewares.robotstxt.RobotsTxtMiddleware',
 'scrapy.downloadermiddlewares.httpauth.HttpAuthMiddleware',
 'scrapy.downloadermiddlewares.downloadtimeout.DownloadTimeoutMiddleware',
 'scrapy.downloadermiddlewares.defaultheaders.DefaultHeadersMiddleware',
 'scrapy.downloadermiddlewares.useragent.UserAgentMiddleware',
 'scrapy.downloadermiddlewares.retry.RetryMiddleware',
 'scrapy.downloadermiddlewares.redirect.MetaRefreshMiddleware',
 'scrapy.downloadermiddlewares.httpcompression.HttpCompressionMiddleware',
 'scrapy.downloadermiddlewares.redirect.RedirectMiddleware',
 'scrapy.downloadermiddlewares.cookies.CookiesMiddleware',
 'scrapy.downloadermiddlewares.httpproxy.HttpProxyMiddleware',
 'scrapy.downloadermiddlewares.stats.DownloaderStats']
2018-12-07 12:15:34 [scrapy.middleware] INFO: Enabled spider middlewares:
['scrapy.spidermiddlewares.httperror.HttpErrorMiddleware',
 'scrapy.spidermiddlewares.offsite.OffsiteMiddleware',
 'scrapy.spidermiddlewares.referer.RefererMiddleware',
 'scrapy.spidermiddlewares.urllength.UrlLengthMiddleware',
 'scrapy.spidermiddlewares.depth.DepthMiddleware']
2018-12-07 12:15:34 [scrapy.middleware] INFO: Enabled item pipelines:
[]
2018-12-07 12:15:34 [scrapy.core.engine] INFO: Spider opened
2018-12-07 12:15:34 [scrapy.extensions.logstats] INFO: Crawled 0 pages (at 0 pages/min), scraped 0 items (at 0 items/min)
2018-12-07 12:15:34 [scrapy.extensions.telnet] DEBUG: Telnet console listening on 127.0.0.1:6023
2018-12-07 12:15:34 [scrapy.downloadermiddlewares.redirect] DEBUG: Redirecting (301) to <GET https://www.reuters.com/robots.txt> from <GET http://www.reuters.com/robots.txt>
2018-12-07 12:15:34 [scrapy.core.engine] DEBUG: Crawled (200) <GET https://www.reuters.com/robots.txt> (referer: None)
2018-12-07 12:15:34 [scrapy.core.engine] DEBUG: Crawled (404) <GET http://www.reuters.com/resources/archive/us/2017.html> (referer: None)
2018-12-07 12:15:34 [scrapy.spidermiddlewares.httperror] INFO: Ignoring response <404 http://www.reuters.com/resources/archive/us/2017.html>: HTTP status code is not handled or not allowed
2018-12-07 12:15:34 [scrapy.core.engine] INFO: Closing spider (finished)
2018-12-07 12:15:34 [scrapy.statscollectors] INFO: Dumping Scrapy stats:
{'downloader/request_bytes': 692,
 'downloader/request_count': 3,
 'downloader/request_method_count/GET': 3,
 'downloader/response_bytes': 30845,
 'downloader/response_count': 3,
 'downloader/response_status_count/200': 1,
 'downloader/response_status_count/301': 1,
 'downloader/response_status_count/404': 1,
 'finish_reason': 'finished',
 'finish_time': datetime.datetime(2018, 12, 7, 17, 15, 34, 400680),
 'httperror/response_ignored_count': 1,
 'httperror/response_ignored_status_count/404': 1,
 'log_count/DEBUG': 4,
 'log_count/INFO': 8,
 'memusage/max': 54824960,
 'memusage/startup': 54824960,
 'response_received_count': 2,
 'scheduler/dequeued': 1,
 'scheduler/dequeued/memory': 1,
 'scheduler/enqueued': 1,
 'scheduler/enqueued/memory': 1,
 'start_time': datetime.datetime(2018, 12, 7, 17, 15, 34, 64757)}
2018-12-07 12:15:34 [scrapy.core.engine] INFO: Spider closed (finished)

I think the problem is that for me, this url does not work: http://www.reuters.com/resources/archive/us/2017.html

Do I need special access rights?  Thanks!

**Comment by zaemyung** [comment]
You're right - it seems Reuters no longer provides archive services with that base url.
Will have to find some other way to crawl.
